src/screens/WasteCollectionScreen.js

```diff
@@ -324,6 +343,45 @@ export const WasteCollectionScreen = ({ navigation }) => {
     setIsDayOverlayVisible(true);
   }, []);
 
+  const triggerExport = useCallback(() => {
+    const { street, zip, city } = getLocationData(streetData);
+
+    const baseUrl = secrets[namespace].serverUrl + staticRestSuffix.wasteCalendarExport;
+
+    let params = `street=${encodeURIComponent(street)}`;
+
+    if (zip) {
+      params += `&zip=${encodeURIComponent(zip)}`;
+    }
+
+    if (city) {
+      params += `&city=${encodeURIComponent(city)}`;
+    }
+
+    const combinedUrl = baseUrl + params;
+
+    if (device.platform === 'android') {
+      Alert.alert(
+        wasteTexts.exportAlertTitle,
+        wasteTexts.exportAlertBody,
+        [
+          {
+            onPress: () => {
+              openLink(combinedUrl);
+            }
+          }
+        ],
+        {
+          onDismiss: () => {
+            openLink(combinedUrl);
+          }
+        }
+      );
+    } else {
+      openLink(combinedUrl);
+    }
+  }, [streetData]);
+
```

Comment:
would be cool to move this in a hook file, to not need to cut and poaste it the next time again from one component to another